content/en/docs/usage/kraft_advanced.md

```diff
@@ -0,0 +1,609 @@
+You can also remove the configuration files by using the `-d, --dist` flags.
+
+### Overview of `kraft clean` command
+
+```text
+Usage: kraft clean [OPTIONS] [LIBS]...
+
+  Clean the build artifacts of a Unikraft unikernel application.
+
+Options:
+  -w, --workdir PATH          Specify an alternative working directory for the
+			      application
+  -p, --proper                Delete the build directory.
+  -d, --dist                  Delete the build directory and configuration
+			      files.
+  --progress / --no-progress  Show progress of build.
+  -h, --help                  Show this message and exit.
+```
+
+## `kraft lib`
+
+If you are maintaining and working directly with Unikraft libraries, you can use `kraft lib`.
+
+### `kraft lib init`
+
+You can initialize a new Unikraft library by using `kraft lib init`.
+You can add the author and version information using `-a, --author-name`, `-e, --author-email` and `-v, --version`.
+When you run the command, you will be able to interactively set all the options for the new library.
+
+```
+$ kraft lib init your-lib
+source (Use $VERSION for automatic versioning):
+project_name [your-lib]:
+lib_name [lib-your-lib]:
+lib_kname [YOURLIB]:
+version: your-first-version
+description []:
+author_name [Your name]:
+author_email [Your email]:
+provide_main [False]:
+with_gitignore [True]:
+with_docs [True]:
+with_patchedir [False]:
+initial_branch [staging]:
+copyright_holder []:
+[INFO    ] Generating files...
+[INFO    ] Generated new library: /home/stefan/projects/unikraft/lib-name
+```
+
+#### Overview of `kraft lib init` command
+
+```text
+Usage: kraft lib init [OPTIONS] [NAME]
+
+  Initialize a new Unikraft library.
+
+Options:
+  -a, --author-name NAME       The author's name for library.
+  -e, --author-email EMAIL     The author's email for library.
+  -v, --version VERSION        Set the known version of the library.
+  -s, --origin TEXT            Source code origin URL.  Use $VERSION in the
+			       URL for automatic versioning.
+  -m, --with-main              Provide a main function override.
+  -w, --workdir PATH           Specify an alternative directory for the
+			       library (default is cwd).
+  -F, --force                  Overwrite any existing files.
+  -q, --no-prompt              Do not prompt for additional data.
+  -S, --soft-pack              Softly pack the component so that it is
+			       available via kraft list.
+  -b, --initial-branch BRANCH  The initial Git branch of the new library.
+  -h, --help                   Show this message and exit.
+
+ ```
+
+### `kraft lib add` and `kraft lib remove`
+
+You can add and remove libraries from your project by using `kraft list add` and `kraft list remove`.
+
+```
+$ kraft list add pthread-embedded
+[INFO    ] Adding lib/pthread-embedded@0.7.0...
+
+$ kraft list remove pthread-embedded
+[INFO    ] Removing lib/pthread-embedded...
+```
+
+#### Overview of `kraft lib add` and `kraft lib remove` commands
+
```

Comment:
Add some text here, saying something like "You can get a full list of features by running `kraft lib add` or `kraft lib remove` with no arguments:"